docs/CubesterYT/KeySimulation.md

```diff
@@ -0,0 +1,43 @@
+# Key Simulation
+
+This extension allows you to simulate Key Presses and Mouse Clicks on the Project.
+
+<hr>
```

Comment:
i tolerated this for window controls because you had to convert the page, but you really ought to be using actual markdown

Reply:
I can change hrs to *** then